tests/test_serializers.py

```diff
@@ -173,6 +177,26 @@ def test_view_project_pointer_count_excludes_folders(self):
         # pointer_project is included in count, but not folder
         assert_equal(result['node']['points'], 1)
 
+    def test_view_project_pending_registration_for_admin_user_does_contain_cancel_link(self):
+        pending_reg = RegistrationFactory(project=self.node, archive=True)
+        assert_true(pending_reg.is_pending_registration)
+        result = _view_project(pending_reg, Auth(self.user))
+
+        assert_not_equal(result['node']['disapproval_link'], '')
+        assert_in('/?token=', result['node']['disapproval_link'])
+        pending_reg.remove()
+
+    def test_view_project_pending_registration_for_admin_user_does_not_contain_cancel_link(self):
```

Comment:
It seems like these tests indicate contradictory behavior. My guess is that means there's a condition that causes it to sometimes contain a link and sometimes not. Be sure that the condition is in the title.